app/javascript/components/patient/Patient.js

```diff
@@ -35,27 +60,45 @@ class Patient extends React.Component {
       this.props.details.address_line_2 ||
       this.props.details.address_city ||
       this.props.details.address_state ||
-      this.props.details.address_zip;
+      this.props.details.address_zip ||
+      this.props.details.address_county;
+    const showMonitoredAddress =
+      this.props.details.monitored_address_line_1 ||
+      this.props.details.monitored_address_line_2 ||
+      this.props.details.monitored_address_city ||
+      this.props.details.monitored_address_state ||
+      this.props.details.monitored_address_zip ||
+      this.props.details.monitored_address_county;
     const showForeignAddress =
       this.props.details.foreign_address_line_1 ||
       this.props.details.foreign_address_line_2 ||
+      this.props.details.foreign_address_line_3 ||
```

Comment:
`this.props.details.foreign_address_state` is missing from this.

Reply:
fixed